etcd/v3/kv_etcd.go

```diff
@@ -733,7 +734,7 @@ func (et *etcdKV) LockWithTimeout(
 			}
 		}
 		if err != nil && time.Since(startTime) > lockTryDuration {
-			return nil, err
+			return nil, errors.New("Lock already taken")
```

Comment:
nit: all golang errors should start with a lower case. Lets also change the error message to:
```
fmt.Errorf("failed to take a lock on %v: lock taken by: %v ", key, currLockerTag.LockerID)
```

Reply:
Sure I'll change that